zulip/integrations/openshift/zulip_openshift_config.py

```diff
@@ -28,6 +30,14 @@ def deployment_notice_destination(branch: str) -> Optional[Dict[str, str]]:
     return None
 
 
+# To utilize Zulip's global times (https://zulip.com/help/global-times)
+# call this function from format_deployment_message, and use the
+# returned string in your message template.
+def get_global_time(dt_str: str) -> str:
+    dt = zulip.datetime_fromisoformat(dt_str.replace("", "T", 1))
```

Comment:
`""` is certainly wrong. Maybe you meant `" "`, but `datetime.fromisoformat` already handles the space ([documentation](https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat)).